Assets/HoloToolkit/Input/Scripts/Utilities/ControllerFinder.cs

```diff
@@ -16,33 +16,42 @@ namespace HoloToolkit.Unity.InputModule
     /// <summary>
     /// ControllerFinder is a base class providing simple event handling for getting/releasing MotionController Transforms
     /// </summary>
-    public class ControllerFinder : MonoBehaviour
+    public abstract class ControllerFinder : MonoBehaviour
     {
-        #region public members
-        public MotionControllerInfo.ControllerElementEnum Element { get { return element; } }
-        public Transform ElementTransform { get { return elementTransform; } private set { elementTransform = value; } }
-        #endregion
+        public MotionControllerInfo.ControllerElementEnum Element
+        {
+            get { return element; }
+            set { element = value; }
```

Comment:
The old version did not have a `set`.

Reply:
Indeed, I added this intentionally.

Reply:
Why? What does changing the `Element` at runtime do?

Reply:
Potentially you could update the logic being performed in the Solver Handler, or change it's position, without having to destroy and create a new instance of the component/object.